homeassistant/components/media_player/__init__.py

```diff
@@ -115,6 +117,38 @@
     ATTR_INPUT_SOURCE,
 ]
 
+# Service call validation schemas
+MEDIA_PLAYER_SIMPLE_SCHEMA = vol.Schema({
+    ATTR_ENTITY_ID: cv.entity_ids,
+})
+
+MEDIA_PLAYER_MUTE_VOLUME_SCHEMA = vol.Schema({
```

Comment:
I'll do it that way because people will copy what they see and as schemas may get more complicated using extend instead of copying the same thing over and over seems better.